### PR TITLE
Added calculateChangedBits()

### PR DESCRIPTION
**Pull request by iddan**

### CODE REVIEW

**Comment by jamiebuilds** [comment]
This is a polyfill of the React API, it will not be adding any additional API or making any changes

**Comment by iddan** [comment]
This is part of the React API

**Comment by iddan** [comment]
https://github.com/facebook/react/blob/29e8924c70856bef9b11e0c74a450140bfcce773/packages/react/src/ReactContext.js#L16

File: src/index.js [hunk 1]
```diff
@@ -70,7 +78,39 @@ function createReactContext<T>(defaultValue: T): Context<T> {
...
+          (oldValue !== oldValue && newValue !== newValue) // eslint-disable-line no-self-compare
+        ) {
+          // No change.
+          changedBits = 0;
+        } else {
+          changedBits =
+            typeof calculateChangedBits === 'function'
+              ? calculateChangedBits(oldValue, newValue)
+              : MAX_SIGNED_31_BIT_INT;
+          if (__DEV__) {
```
Comment (jamiebuilds) [inline src/index.js]: Can you just use `process.env.NODE_ENV !== 'production'` and get rid of all the other stuff to work around `__DEV__`

**Comment by iddan** [comment]
Ready

**Comment by iddan** [comment]
Ready

**Pull request merged**